src/app/storage/utils.ts

```diff
@@ -0,0 +1,68 @@
+import axios, { AxiosError, AxiosResponse, Method } from 'axios';
+
+import { INITIAL_STORAGE } from './constants';
+import { PatchInstruction } from './types/json-patch';
+import { OverallStorageObject, StorageResponse, StoredTypeNames } from './types/storage';
+
+const token = `
+eyJraWQiOiIyZDM3OTRlOC1iMmJhLTMxYWItYTVmNC0xMjExNDZiZGUxZmMiLCJhbGciOiJSUzI1NiJ9.eyJ0b2tlbktpbmQiOiJiZWFyZXIiLCJqdGkiOiJkMmQzMzUxMC03MzgyLTRhZGMtOWExZi01ZDE1YmVlZmU5NmYiLCJhdWQiOlsiaHR0cDovL2xvY2FsaG9zdDozMDAwIiwiaHR0cHM6Ly93YWxsZXQuc3RhZ2luZy5kbmFzdGFjay5jb20iXSwiYXpwIjoib2ljci1jbGllbnQiLCJpYXQiOjE2MzY2NjEwMzEsImV4cCI6MTYzNjc0NzQzMSwic3ViIjoiNzQyMTY1ZTUtNWZkMy00ZmEwLTkwYmUtODUyZGFiZjY3ODlkIiwiaXNzIjoiaHR0cHM6Ly93YWxsZXQuc3RhZ2luZy5kbmFzdGFjay5jb20iLCJzY29wZSI6InVzZXItc3RvcmFnZSBvcGVuaWQgb2ZmbGluZV9hY2Nlc3MiLCJhY3Rpb25zIjp7fX0.OLjss1PsvF4SW2_KbF0JmGzRlCWexq42Sej-MT5oM1CsA2ZYaP0F0z0nTQSYLEGoxnUcPkWokkT2vW4ynIm96aeu9TOb-SB4iD1VINT6LD3hLGdZKpcsipb1e3sFc0y-waZPt4A-MDizcWPfMZVyjqX6N4EIAuNyqSGwuBswI73Cm_4OnEUuoX7_JamPsE9_fP3LuZSg26bvR9GBouW6QnpqZPEevzFmCorVx34ValNW046-KalhtIdo6dmaJIZdPKcYvrzBK13vxU5RjxeJfqtmu2gymWn_f-nayROijRSQ9c2LqvczuufnnTuM5U9aL3qT3ZLURhqUWIpprmf_6Q
```

Comment:
yep. hardcoded my token, because the fetching code will have to be integrated with @anncatton's stuff anyway, so this here is all up in the air for now.

Reply:
i think better not to merge this token or the wallet url into develop, just use them locally. i've added a config in my auth pr so we can get the urls and client stuff set up - obvs we'll need to do some adjustments to get both our features working together

Reply:
agreed! I'll amend mine once yours is merged!